xml/System.Text.Json/JsonProperty.xml

```diff
@@ -113,10 +139,24 @@
         <Parameter Name="text" Type="System.String" />
       </Parameters>
       <Docs>
-        <param name="text">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="text">The text to compare against.</param>
+        <summary>Compares <paramref name="text" /> to the name of this property.</summary>
+        <returns><see langword="true" /> if the name of this property matches <paramref name="text" />,
+              <see langword="false" /> otherwise.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+This method is functionally equal to doing an ordinal comparison of `text` and <xref:System.Text.Json.JsonProperty.Name>, but can avoid creating the string instance.
```

Comment:
@carlossanlop I don't think this overload avoids the string instance.
@ahsonkhan can you confirm?
```suggestion
This method is functionally equal to doing an ordinal comparison of `text` and <xref:System.Text.Json.JsonProperty.Name>.
```